vars/openShift.groovy

```diff
@@ -2,7 +2,11 @@
 
 
 def withNode(Map parameters = [:], Closure body) {
-    image = parameters.get('image', pipelineVars.defaultNodeImage)
+    jenkinsSlaveImage = parameters.get(
+        'namespace',
+        cloud.equals(pipelineVars.defaultUICloud) ? pipelineVars.centralCIjenkinsSlaveImage : pipelineVars.jenkinsSlaveImage
```

Comment:
This is the offending line. Need to define `jenkinsSlaveImage` after you define `cloud`